api/NuageAPI_test.go

```diff
@@ -278,72 +282,83 @@ func TestVMCreateDelete(t *testing.T) {
 		t.Fatal("Unable to create a test VM")
 	}
 
-	t.Logf("Waiting for 60 seconds before verifying port got resolved with an IP address in VRS")
-	time.Sleep(time.Duration(60) * time.Second)
+	// Registering for OVSDB updates instead of random sleep
+	portInfoUpdateChan := make(chan *PortIPv4Info)
```

Comment:
This block of code is being repeated in almost all test cases. Please create a function and call it in all of the test functions.